### PR TITLE
ENQT-130: Integration - UX Components - Web Dynamic Form - Create Web Dynamic Form Page

### PR DESCRIPTION
**Pull request by FabioFortini**

### CODE REVIEW

File: plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/config/entryMessageType.jsp [hunk 1]
```diff
@@ -80,48 +108,80 @@
                 <div class="panel-body"><s:text name="label.bodyModel.help" /></div>
             </div>   
  --%>                     
+            <%--<div class="form-group">--%>
+                <%--<div class="checkbox">--%>
+                    <%--<wpsf:checkbox id="jpwebdynamicform_store" name="store" cssClass="radiocheck" />&#32;--%>
+                    <%--<label for="jpwebdynamicform_store"><s:text name="label.local.message.store" /></label>--%>
+                <%--</div>--%>
+            <%--</div>--%>
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/config/entryMessageType.jsp]: please delete commented code.

File: plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/config/entryMessageType.jsp [hunk 2]
```diff
@@ -133,12 +193,23 @@
 
             <%-- Indica se si vuole far notificare vial mail il messaggio agli operatori
             Se true, influenzerà la validazione (obbligatorietà) degli altri campi. --%>
+            <%--<div class="form-group">--%>
+                <%--<div class="checkbox">--%>
+                    <%--<wpsf:checkbox id="jpwebdynamicform_notifiable" name="notifiable" />&#32;--%>
+                    <%--<label for="jpwebdynamicform_notifiable"><s:text name="label.automatic.notification.active" /></label>--%>
+                <%--</div>--%>
+            <%--</div>--%>
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/config/entryMessageType.jsp]: please delete commented code

File: plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/entryAnswer.jsp [hunk 3]
```diff
@@ -1,78 +1,76 @@
...
+            <div class="form-group">
+                <div class="col-xs-12">
+                    <s:submit type="button" cssClass="btn btn-primary pull-right">
+                        <s:text name="%{getText('label.send')}"/>
+                    </s:submit>
+                </div>
+            </div>
+        </fieldset>
+    </s:form>
 </div>
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/entryAnswer.jsp]: missing EOF newline

File: plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/messageFinding.jsp [hunk 4]
```diff
@@ -3,295 +3,381 @@
...
+                                    <%-- from --%>
                                 <div class="form-group">
-                                    <label class="control-label col-sm-2 text-right" for="<s:property value="#currentFieldId" />_end_cal"><s:property value="#attribute.name" />&#32;<s:text name="End"/></label>
+                                    <label for="jpwebdynamicform_from_cal"
+                                           class="control-label col-sm-2 text-right"><s:text name="label.from"/></label>
                                     <div class="col-sm-5">
-                                        <wpsf:textfield id="%{#currentFieldId}_end_cal" cssClass="form-control datepicker" name="%{#dateEndInputFieldName}" value="%{getSearchFormFieldValue(#dateEndInputFieldName)}" />
-                                        <span class="help-block">dd/MM/yyyy</span>
+                                        <wpsf:textfield name="from" id="jpwebdynamicform_from_cal"
+                                                        cssClass="form-control"/>
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/messageFinding.jsp]: fix formatting

File: plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/messageFinding.jsp [hunk 5]
```diff
@@ -3,295 +3,381 @@
...
+                            <wpsf:hidden name="%{#numberEndInputFieldName}"
+                                         value="%{getSearchFormFieldValue(#numberEndInputFieldName)}"/>
+                        </s:elseif>
+                        <s:elseif test="#attribute.type == 'Boolean'">
+                            <%-- DA FARE --%>
+                        </s:elseif>
+                        <s:elseif test="#attribute.type == 'ThreeState'">
+                            <%-- DA FARE --%>
+                        </s:elseif>
+                    </s:iterator>
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/messageFinding.jsp]: what's left to do?

File: plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/trashMessage.jsp [hunk 6]
```diff
@@ -1,27 +1,68 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/trashMessage.jsp]: please remove extra new lines

File: plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/messageFinding.jsp (hunk 5 above)
Comment (FabioFortini) [inline plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/messageFinding.jsp]: Per quanto riguarda questi commenti, erano già presenti al momento della mia presa in carico della carta.

File: plugins/entando-plugin-jpwebdynamicform/src/main/java/com/agiletec/plugins/jpwebdynamicform/apsadmin/message/package_it.properties [hunk 7]
```diff
@@ -44,13 +48,13 @@ admin.jpwebdynamicform.title.messageManagement=Gestione Messaggi
 admin.jpwebdynamicform.Config.edit=Configura Mail
 
 #VALIDATION LABELS
-requiredstring=${getText(fieldName)} � richiesto.
+requiredstring=${getText(fieldName)} � richiesto.
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpwebdynamicform/src/main/java/com/agiletec/plugins/jpwebdynamicform/apsadmin/message/package_it.properties]: issues again with characters encodings.
Guys please check your IDE behaviour.

File: plugins/entando-plugin-jpwebdynamicform/src/main/java/com/agiletec/plugins/jpwebdynamicform/apsadmin/message/package_it.properties [hunk 8]
```diff
@@ -44,13 +48,13 @@ admin.jpwebdynamicform.title.messageManagement=Gestione Messaggi
...
 
 Errors.file.wrongFileNameFormat=Il nome del File ''{0}'' contiene caratteri non consentiti. Sono permessi caratteri alfanumerici senza spazi.
 
 Message.eMailAddress.notFound=Impossibile inviare una risposta vial eMail. Indirizzo eMail non trovato.
 Message.message.notFound=messaggio non trovato.
-Errors.answer.sendingError=Si � verificato un errore durante l'invio della notifica. Riprovare tra un po'
+Errors.answer.sendingError=Si � verificato un errore durante l'invio della notifica. Riprovare tra un po'
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpwebdynamicform/src/main/java/com/agiletec/plugins/jpwebdynamicform/apsadmin/message/package_it.properties]: issues again with characters encodings.

File: plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/messageFinding.jsp (hunk 5 above)
Comment (nicpuddu) [inline plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/messageFinding.jsp]: ok, we'll leave it there for now and open another issue, thx

**Comment by nicpuddu** [comment]
:shipit:

**Pull request merged**